magento/magento2
# Fix incorrect less escape

### Description (*)

Small changes correct less escape for width element

Before fix can lead to such case

output
width: calc(100% - 13px) <= incorrect value we intend to use

After fix

output
width: calc(100% - 33px) <= correct value
width: calc(100% - (23px + 10px)) 

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. Enable date of birth field for customer (require option => optional or required)
2. Go to customer create account at frontend page
3. Check visual UI date input
4. Compare both case static content deploy php vs grunt less compile

### Questions or comments


### Contribution checklist (*)
